Update DAG example links in multiple providers documents

amirmor1 opened the pull request:
<!--
 Licensed to the Apache Software Foundation (ASF) under one
 or more contributor license agreements.  See the NOTICE file
 distributed with this work for additional information
 regarding copyright ownership.  The ASF licenses this file
 to you under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance
 with the License.  You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing,
 software distributed under the License is distributed on an
 "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->

<!--
Thank you for contributing! Please make sure that your code changes
are covered with tests. And in case of new features or big changes
remember to adjust the documentation.

Feel free to ping committers for the review!

In case of an existing issue, reference it using one of the following:


related: #ISSUE

How to write a good git commit message:
http://chris.beams.io/posts/git-commit/
-->

closes: #44012 


<!-- Please keep an empty line above the dashes. -->
---

Updated multiple provider documents with new DAG example links.

amirmor1 committed:
44012 - Update index.rst

amirmor1 committed:
Merge pull request #2 from amirmor1/amirmor1-44012-fix-docs

44012 - Update index.rst

amirmor1 committed:
Update example DAG link in apache-airflow-providers-docker/index.rst

Resolves: #4402

amirmor1 committed:
Update index.rst

eladkal commented on docs/apache-airflow-providers-docker/index.rst:
```diff
@@ -49,7 +49,7 @@
     :caption: Resources
 
-    Example DAGs <https://github.com/apache/airflow/tree/providers-docker/|version|/providers/tests/system/docker>
+    Example DAGs <https://github.com/apache/airflow/tree/providers-docker/|version|/tests/system/providers/docker>
```
This seems to be wrong for all providers.
can you apply a fix for all?

amirmor1 replied:
Done

Commit:
Update example dag links in multiple providers documents

Any provider document that was not changed either does not have DAG example provided before or their link works.

Resolves: #4402

Commit:
Merge branch 'main' of github-amirmor1:amirmor1/airflow into patch-1

Commit:
Merge branch 'patch-1' of github-amirmor1:amirmor1/airflow into patch-1

amirmor1 committed:
Merge branch 'main' into patch-1

kaxil commented on docs/apache-airflow-providers-airbyte/index.rst:
```diff
@@ -55,7 +55,7 @@
     :caption: Resources
 
-    Example DAGs <https://github.com/apache/airflow/tree/providers-airbyte/|version|/providers/tests/system/airbyte>
+    Example DAGs <https://github.com/apache/airflow/tree/providers-airbyte/|version|/tests/system/providers/airbyte>
```
There is no such link though `https://github.com/apache/airflow/tree/providers-airbyte/`

amirmor1 replied:
This link seems to be working. What am I missing here?
https://github.com/apache/airflow/tree/providers-airbyte/4.0.0/tests/system/providers/airbyte

kaxil replied:
aah my bad
